### PR TITLE
Add Java Solution

### PR DESCRIPTION
**Pull request by NewoIsTaken**

### CODE REVIEW

File: solutions/USACO dec16-moocast-silver.mdx [hunk 1]
```diff
@@ -111,4 +111,78 @@ print(ans)
 
 </PySection>
 
+<JavaSection>
+
+```java
+import java.io.File;
+import java.io.PrintWriter;
+import java.util.Scanner;
```
Comment (andrewwangva) [inline solutions/USACO dec16-moocast-silver.mdx:119]: > import java.io.File;
import java.io.PrintWriter;
import java.util.Scanner;

import java.io.*;
import java.util.*;
is a lot cleaner

Comment (andrewwangva) [reply solutions/USACO dec16-moocast-silver.mdx:119]: `import java.io.*;
import java.util.*;
`

some reason the star got deleted :(

**Review by andrewwangva — COMMENTED** [review]
Everything else looks good. it'd be nice if you deleted some of the extra spacing and used a [Buffered Reader](https://usaco.guide/general/io?lang=java) instead of a Scanner to speed up the input.

**Comment by NewoIsTaken** [comment]
OK, I will edit it tonight.

File: solutions/USACO dec16-moocast-silver.mdx (hunk 1 above)
Comment (NewoIsTaken) [reply solutions/USACO dec16-moocast-silver.mdx]: Am using `*` now.

**Comment by NewoIsTaken** [comment]
@andrewwangva Ah, was going to do that but thanks!

**Pull request merged**